api/XRPermissionStatus.json

```diff
@@ -1,102 +0,0 @@
-{
-  "api": {
-    "XRPermissionStatus": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/XRPermissionStatus",
```

Comment:
https://developer.mozilla.org/docs/Web/API/XRPermissionStatus will need to be removed.

Reply:
It still exists in the spec and there is a tracking issue https://bugzilla.mozilla.org/show_bug.cgi?id=1582344, so maybe it should remain? 🤔

Reply:
Is it shipped anywhere yet?

Reply:
Nah, but there have been requests to keep things that authors want to eventually ship, https://github.com/mdn/browser-compat-data/pull/6954#issuecomment-723939897 for example. #10619 is relevant here I guess.